config/locales/en.yml

```diff
@@ -0,0 +1,28 @@
+en:
+  servactory:
+    input_arguments:
+      checks:
+        inclusion:
+          default_error: "[%{service_class_name}] Wrong value in `%{input_name}`, must be one of `%{input_inclusion}`"
+        must:
+          default_error: "[%{service_class_name}] Input `%{input_name}` must \"%{code}\""
```

Comment:
There are no tests for this.

Reply:
The error is blocked by another error that is ahead. It is impossible to catch it at the current moment, but it is present. Example from `Usual::Example7`:

```
#<Servactory::InputArguments::Tools::CheckErrors:0x0000000103f53170 @collection=["[Usual::Example7] Wrong type in input array `invoice_numbers`, expected `String`", "[Usual::Example7] Input `invoice_numbers` must \"be_6_characters\""]>
```